main/src/ca/uwaterloo/flix/language/errors/NameError.scala

```diff
@@ -234,4 +234,17 @@ object NameError {
     }
   }
 
+  case class MismatchedTypeParamKinds(name: String, loc1: SourceLocation, kind1: Kind, loc2: SourceLocation, kind2: Kind) extends NameError {
+    def summary: String = "Mismatched type parameter kinds."
+    def message: VirtualTerminal = {
+      val vt = new VirtualTerminal
+      vt << Line(kind, source.format) << NewLine
+      vt << ">> Mismatched kinds for type parameter '" << Red(name) << "'." << NewLine
+      vt << NewLine
+      vt << Code(loc1, s"Kind: $kind1") << NewLine // TODO might need FormatKind
```

Comment:
TODO

Reply:
I'll remove this for now. If kind vars need to be displayed to the user, we'll need a format kind so display can vary by audience.